samples/wifi/softap/README.rst

```diff
@@ -0,0 +1,136 @@
+.. _wifi_softap_sample:
+
+Wi-Fi: SoftAP
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The SoftAP sample demonstrates how to start a nRF70 Series device in Software-enabled Access Point (SoftAP or SAP) mode.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample enables SoftAP mode and allows the devices to connect to it.
+It starts a Dynamic Host Configuration Protocol (DHCP) server to assign the automatic IP address to the connected devices.
+Then, the sample lists the connected devices.
+
+You can enable the SoftAP mode by setting the below configuration options in the sample project configuration file:
+
+* :kconfig:option:`CONFIG_NRF700X_AP_MODE`: Enables access point mode support.
+* :kconfig:option:`CONFIG_WPA_SUPP_AP`: Enables access point support.
+
+.. note::
+
+   The SoftAP mode operation is dictated by regulatory requirements.
+
+You can set the regulatory domain using the below configuration option:
+
+* :kconfig:option:`CONFIG_SOFTAP_SAMPLE_REG_DOMAIN`: Sets the ISO/IEC alpha2 country code.
+
+For more information, see the :ref:`nRF70_soft_ap_mode` documentation.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following Kconfig options are used in this sample (located in :file:`samples/wifi/softap/Kconfig`):
+
+.. options-from-kconfig::
+
+IP addressing
+*************
+
+The sample starts the DHCP server on the SoftAP interface.
+The station devices should use DHCP to get an IP address from the virtual router.
+
+To specify the DHCP pool start address, you can edit the following configuration option in the :file:`prj.conf` file:
```

Comment:
This does not need to be in a code-block? Since it is only 1 option.
```suggestion
To specify the DHCP pool start address, you can edit `` CONFIG_SOFTAP_SAMPLE_DHCPV4_POOL_START="192.168.1.2"`` option in the :file:`prj.conf` file:
```